tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_advertiser.c

```diff
@@ -51,6 +55,58 @@ static void adv_init(void)
 	ASSERT_OK(bt_mesh_adv_enable(), "Mesh adv init failed");
 }
 
+static void allocate_all_net_buf(struct net_buf **buf, size_t num_buf, uint8_t xmit)
```

Comment:
Suggestion (up to you, but I find current names confusing):
`allocate_all_net_buf` -> `allocate_adv_array`
`send_all_adv_buf` -> `send_adv_array`